extensions/shared/external-media/sources/google-photos/filter-view.js

```diff
@@ -0,0 +1,77 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { Fragment, useState } from '@wordpress/element';
+import { SelectControl, Button } from '@wordpress/components';
+
+const FILTERS = [
+	{ label: __( 'Category', 'jetpack' ), value: 'category' },
+	{ label: __( 'After date', 'jetpack' ), value: 'startDate' },
+	{ label: __( 'Before date', 'jetpack' ), value: 'endDate' },
+	{ label: __( 'Favourite Media Only', 'jetpack' ), value: 'favorite' },
```

Comment:
```suggestion
	{ label: __( 'Favorite Media Only', 'jetpack' ), value: 'favorite' },
```

Reply:
I'd even suggest using the same wording as in the added filter - "Only favorites":
   <img width="215" alt="Screenshot 2020-05-21 13 55 23" src="https://user-images.githubusercontent.com/1451471/82556858-d922da00-9b6a-11ea-94ca-bb13a6f8c259.png">